docs/fundamentals/code-analysis/style-rules/ide0018.md

```diff
@@ -26,7 +26,7 @@ dev_langs:
 
 ## Overview
 
-This style rule concerns whether `out` variables are declared inline or not. Starting in C# 7, you can [declare an out variable in the argument list of a method call](../../../csharp/language-reference/keywords/out-parameter-modifier.md#calling-a-method-with-an-out-argument), rather than in a separate variable declaration.
+This style rule concerns whether `out` variables are declared inline or not. Starting in C# 7, you can [declare an out variable in the argument list of a method call](/dotnet/csharp/language-reference/keywords/method-parameters#out-parameter-modifier#calling-a-method-with-an-out-argument), rather than in a separate variable declaration.
```

Comment:
```suggestion
This style rule concerns whether `out` variables are declared inline or not. Starting in C# 7, you can [declare an out variable in the argument list of a method call](../../../csharp/language-reference/keywords/method-parameters.md#out-parameter-modifier), rather than in a separate variable declaration.
```